tests/utils/nodes/nodes.go

```diff
@@ -2,19 +2,27 @@ package nodes
 
 import (
 	"context"
+	"encoding/json"
 	"time"
 
 	"github.com/golang/glog"
 	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/client"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-// WaitForNodesReady waits for all nodes become ready.
-func WaitForNodesReady(cs *client.ClientSet, timeout, interval time.Duration) error {
+type patchStringValue struct {
```

Comment:
The name of this struct doesn't make sense. Struct can't be an action. Can you please rename ?